rasa/importers/skill.py

```diff
@@ -0,0 +1,188 @@
+import logging
+from functools import reduce
+from typing import Text, Set, Dict, Optional, List, Union, Any
+import os
+
+from rasa import data
+import rasa.utils.io as io_utils
+from rasa.core.domain import Domain
+from rasa.core.interpreter import RegexInterpreter, NaturalLanguageInterpreter
+from rasa.core.training.dsl import StoryFileReader
+from rasa.importers.importer import TrainingFileImporter
+from rasa.importers import utils
+from rasa.nlu.training_data import TrainingData
+from rasa.core.training.structures import StoryGraph
+
+logger = logging.getLogger(__name__)
+
+
+class SkillSelector(TrainingFileImporter):
+    def __init__(
```

Comment:
Just for my understanding: All the parameters need to be set in the config file?

Reply:
it tries to pass in `config_path`, `domain_path`, `training_data_paths` which are given through the cli, the rest has to be provided through the config file.

Reply:
ok, I guess I missed that part, will have another look